src/app/onion/learning-object-builder/components/info-page/metadata/metadata.component.html

```diff
@@ -13,7 +13,34 @@
       <input class="name-field" type="text" name="name" placeholder="Learning Object Name" [(ngModel)]="learningObject.name"
         #name="ngModel" [ngClass]="{'bad': errorStore.error === 'name' }" [pattern]="validName" />
     </div>
-
+    <div class="input-group">
+        <div class="label">
+          Add more authors.
+        </div>
+        <div *ngIf="selectedAuthors.length > 0">
+          <div class="level-list">
+            <div class="level button" *ngFor="let user of selectedAuthors; let i = index;"
+              (click)="toggleLevel(level)">
+              {{ user }}<span (click)="removeAuthor(user)"><i class="fa fa-trash" aria-hidden="true"></i></span>
+            </div>
+          </div>
+        </div>
+        <input #userSearchInput autofocus type="text" class="search-input" placeholder="Search all users" [(ngModel)]="query.text">
+        <div id="container">
+            <ul>
+              <li *ngFor="let user of arrayOfKeys; let i = index;">
+                <span *ngIf="this.isAuthorSelected(i)" id="plus" (click)="addAuthor(i)">
+                  <i class="fas fa-plus"></i>
+                </span>
+                <span *ngIf="!this.isAuthorSelected(i)" id="delete" (click)="removeAuthor(users[user]._username)">
+                    <i class="fa fa-trash"></i>
+                </span>
+                <span class="navbar-gravatar" [ngStyle]="{'background-image': 'url(' + userService.getGravatarImage(users[user]._email, 200) + ')'}"></span>
```

Comment:
Should be able to use `.email` getter method if object is instantiated as a User entity. Best to keep consistency in the way we're accessing properties on entities.